frontend/control-center/src/pages/Connectors/ConfigureConnector/SetConfigInputs/SetConfigInputs.tsx

```diff
@@ -0,0 +1,61 @@
+import {Input} from 'components';
+import {Source} from 'model';
+import React, {Dispatch, SetStateAction, useEffect, useState} from 'react';
+import {useTranslation} from 'react-i18next';
+import styles from './SetConfigInputs.module.scss';
+
+type SetConfigInputsProps = {
+  configurationValues: {[key: string]: string};
+  setConfig: Dispatch<SetStateAction<{}>>;
+  storedConfig: {};
+  source: string;
+};
+
+export const SetConfigInputs = (props: SetConfigInputsProps) => {
+  const {configurationValues, storedConfig, source} = props;
+  const [input, setInput] = useState(storedConfig || configurationValues);
+  const {t} = useTranslation();
+  const inputArr: any = [];
+
```

Comment:
I guess this would work for the type instead of any: 
  
`  const inputArr: ReactElement[] = [];`